misc/src/main/java/com/example/snippets/backgroundwork/WakeLockSnippetsJava.java

```diff
@@ -10,6 +10,7 @@
 public class WakeLockSnippetsJava extends Activity {
 
     PowerManager.WakeLock wakeLock;
+    long wlTimeout = 10*60*1000L; // 10 minutes
```

Comment:
Suggest using const casing for this and making it a const 
eg.`WAKELOCK_TIMEOUT`